content/en/docs/setup/install/helm/index.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Customizable Install with Helm
+description: Install and configure Istio for in-depth evaluation or production use.
+weight: 27
+keywords: [kubernetes,helm]
+owner: istio/wg-environments-maintainers
+icon: helm
+test: no
+---
+
+Follow this guide to install and configure an Istio mesh using
+[Helm](https://helm.sh/docs/) for in-depth evaluation or production use.
+
+The Helm charts used in this guide are the same underlying charts used when
+installing Istio via [Istioctl](/docs/setup/install/istioctl/) or the
+[IstioOperator API](/docs/setup/install/operator/).
+
+## Prerequisites
+
+1. [Download the Istio release](/docs/setup/getting-started/#download).
+
+1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/).
+
+1. Check the [Requirements for Pods and Services](/docs/ops/deployment/requirements/).
+
+1. [Install a Helm client](https://helm.sh/docs/intro/install/) with a version higher than 3.1.
+
+    {{< warning >}}
+    Use a 3.x version of Helm. Helm 2 is not supported.
+    {{< /warning >}}
+
+The commands in this guide use the Helm charts that are included in the Istio release packages.
+
+## Installation steps
+
+Change directory to the root of the release and then
+follow the instructions below.
+
+1. Create a namespace for the `istio-system` components:
+
+    {{< text bash >}}
+    $ kubectl create namespace istio-system
+    {{< /text >}}
+
+1. Install the Istio base chart which contains cluster level resources like
+   Custom Resource Definitions (CRDs), cluster role and cluster role bindings
+   and service accounts used by the Istio components:
+
+    {{< text bash >}}
+    $ helm install --namespace istio-system istio-base manifests/charts/base
+    {{< /text >}}
+
+1. Install the Istio discovery (istiod) chart which includes the Istio control
+   plane deployment and its associated configuration:
+
+    {{< warning >}}
+    The default chart configuration uses the secure third party tokens for Service
+    Account token projections used by Istio proxies to authenticate with the Istio
+    control plane. Before proceeding to install this chart, you should verify
+    if third party tokens are enabled in your cluster by following the steps
+    describe [here](/docs/ops/best-practices/security/#configure-third-party-service-account-tokens).
+    If third party tokens are not enabled, you should add the option
+    `--set global.jwtPolicy=first-party-jwt` to the following command.
+    If the `jwtPolicy` is not set correctly, the `istiod` pod will not get
+    deployed due to the missing `istio-token` volume.
+    {{< /warning >}}
+
+    {{< text bash >}}
+    $ helm install --namespace istio-system istiod manifests/charts/istio-control/istio-discovery
+    {{< /text >}}
+
+1. (Optional) Install the Istio ingress gateway chart which contains the ingress
+   gateway components:
+
+    {{< warning >}}
+    Similar to the discovery chart above, the default ingress gateway chart uses
```

Comment:
could this be summarized to not repeat the full text? here and below.

Reply:
I was going to but wanted to add warnings again for folks who revisit the page and just Helm install egress/ingress GW. Since the UX is horrible if you miss the JWT settings as compared to istioctl which switches based on your environment, I leaned towards repetition. Let me see if I can reduce the common text.